README.md

```diff
@@ -58,62 +58,45 @@ help = result.replace("Usage\n\n", "").rstrip()
 cog.out(f"```\n{help}\n```")
 ]]] -->
 ```
-
```

Comment:
Please revert those changes that do not concern the Cider package.

Reply:
I've removed these lines - didn't see them, even though I scrolled to the line(s) in question. Probably human error. I've removed them. I think this is a good case for my proposal to only update the README once, just prior to publishing a new revision. It might've mitigated the go-backs. That said, in this case, it was my error but this might happen again for someone else.